nservicebus/service-fabric/sagas.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Service Fabric Persistence Sagas
+reviewed: 2017-03-22
+component: ServiceFabricPersistence
+---
+
+## Reliable collections
+
+When using the Service Fabric Persistence with a reliable service, saga data is stored in reliable dictionaries.  
+
+## Saga data serialization
+
+Service Fabric Persistence stores saga data in JSON format using uses Json.NET. 
```

Comment:
"Saga data is stored in JSON format using uses Json.NET. "

and link to json.net